package.json

```diff
@@ -11,11 +11,13 @@
   ],
   "dependencies": {},
   "devDependencies": {
```

Comment:
we should add in es5-shim, unless we want to be super careful with which features of babel we use that compile down to ES3.